data/net/minecraft/world/item/alchemy/PotionUtils.mapping

```diff
@@ -1,38 +1,68 @@
 CLASS net/minecraft/world/item/alchemy/PotionUtils
 	METHOD addPotionTooltip (Lnet/minecraft/world/item/ItemStack;Ljava/util/List;F)V
-		ARG 0 item
-		ARG 1 lores
+		COMMENT Adds the tooltip of the {@code Potion} stored on the {@code ItemStack} along a "durationFactor"
+		ARG 0 stack
+			COMMENT the passed {@code ItemStack}
+		ARG 1 tooltips
+			COMMENT the passed List of current {@code Component} tooltips
 		ARG 2 durationFactor
+			COMMENT the passed durationFactor of the {@code Potion}
 	METHOD getAllEffects (Lnet/minecraft/nbt/CompoundTag;)Ljava/util/List;
-		COMMENT Creates a list of PotionEffect from data on a NBTTagCompound.
-		ARG 0 tag
+		COMMENT Creates a List of {@code MobEffectInstance}s from data on a {@code CompoundTag}.
```

Comment:
Capitalised L in list referring to class should use `@code` or `@link`. There are also a few discrepancies in this class related to this (some lists begin with a capital L whilst others don't).